third_party/blink/renderer/core/brave_page_graph/graph_item/edge/js/edge_js_call.h

```diff
@@ -22,14 +21,14 @@ class EdgeJSCall final : public EdgeJS {
   EdgeJSCall(GraphItemContext* context,
              NodeScript* out_node,
              NodeJS* in_node,
-             const std::vector<std::string>& arguments,
+             WTF::Vector<WTF::String> arguments,
```

Comment:
I guess here we should also just write `Vector<String>`

Reply:
Done